src/sentry/models/tombstone.py

```diff
@@ -26,7 +26,6 @@ class TombstoneBase(Model):
 
     class Meta:
         abstract = True
-        unique_together = ("table_name", "object_identifier")
```

Comment:
Was this ignored because the subclasses define their own Meta?

Reply:
Yep, that appears to be the case. It's a bit unfortunate since this would ideally be a unique index, but I'm not sure it's worth the effort deduplicating rows to accomplish this now 😞